pkgs/tools/misc/wasm-tools/default.nix

```diff
@@ -0,0 +1,39 @@
+{ rustPlatform
+, fetchFromGitHub
+, lib
+, stdenv
```

Comment:
```suggestion
{ lib
, rustPlatform
, fetchFromGitHub
, stdenv
```

Reply:
> [Arguments should be listed in the order they are used, with the exception of `lib`, which always goes first.](https://nixos.org/manual/nixpkgs/unstable/#sec-syntax)

Reply:
Ouch. I am a maintainer of some packages and I just learnt about it, sorry. Updated, thanks!

Reply:
No problem, this is kind of hidden in blocks of text, understandable